packages/demo/basic/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@venncity/graphql-demo-basic",
+  "version": "1.0.0",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "Venn Engineering",
+  "description": "a basic demo project showcasing the usage of the graphql-clou packages",
+  "config": {
+    "db_port": 5432
+  },
+  "scripts": {
+    "build": "yarn build:cjs && yarn build:esm",
+    "build:cjs": "rm -rf lib/cjs && tsc",
+    "build:esm": "rm -rf lib/esm && tsc -m esnext --outDir lib/esm",
+    "postinstall": "patch-package",
+    "lint": "eslint . --fix  --ext .js,.ts",
+    "lint:ci": "eslint . --format junit -o $LINT_JUNIT_OUTPUT",
+    "test": "echo TODO",
+    "test:ci": "echo TODO",
```

Comment:
why we need this ci commands here?

Reply:
Coz it's part of the lerna structure and without these cmds the build will fail